components/ucd/normal/tests/conformance_tests.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+
+extern crate unic_ucd_normal;
+extern crate unic_ucd_utils;
+
+use unic_ucd_normal::DecompositionType as DT;
+use unic_ucd_utils::iter_all_chars;
+use std::collections::HashSet as Set;
```

Comment:
In UNIC, I'm trying to keep `use` statements in four groups, split with one empty line: 1) core/std lib, 2) third-party, 3) this workspace, and 4) this package. (I should write the style guide soon, it's not anywhere yet.)

And another nit: usually we don't rename unless it's useful or necessary. Using original name, `HashSet`, would be fine here.